src/lib/Post.svelte

```diff
@@ -72,22 +77,42 @@
 		<button 
 			class="pfp" 
 			on:click={()=>{
-				profileClicked.set(post.user);
+				if (post.user == "Discord") {
```

Comment:
why don't you also set `post.user` to the bridged user?

Reply:
?

Reply:
In the other part of showing the post where it sets `userName` to the bridged user, why not also set `post.user` to it?

Reply:
yeah im stupid for not knowing